src/pages/docs/[source]/[tag]/search.vue

```diff
@@ -32,11 +79,21 @@ import { useRoute } from 'vue-router';
 
 import { search, DocumentType } from '~/util/search';
 
+const filter = ref([
```

Comment:
Since it's not holding a primitive, this should use `reactive` instead.

ref: https://stackoverflow.com/a/65262638/6407023